components/onboarding-modal/OnboardingContentBox.js

```diff
@@ -47,9 +47,9 @@ class OnboardingContentBox extends React.Component {
   }
 
   componentDidMount() {
-    const member = this.props.LoggedInUser.memberOf.filter(member => member.collective.id === this.props.collective.id);
+    const member = this.props.LoggedInUser.memberOf.find(member => member.collective.id === this.props.collective.id);
     this.setState({
-      admins: [{ role: 'ADMIN', member: this.props.LoggedInUser.collective, id: member[0].id }],
```

Comment:
I don't think this will fix https://github.com/opencollective/opencollective/issues/3130, if the member doesn't exist `find` will return `null` so `member.id` will still crash

Reply:
I'm not quite sure how to approach it then... the one idea I had was, if the `member.id` is not found for the creator/admin of the collective, then make it `null` during the onboarding flow. Then when they click 'submit', in the prep before we send the `EditCoreContributors` mutation we try to find the `member.id` one more time? Because if we submit `EditCoreContributors` without the creator admin having `member.id`, it thinks that we're trying to remove them as admin of the collective and the mutation won't go through.

Reply:
The suggestion from @kewitz below (https://github.com/opencollective/opencollective-frontend/pull/5460#discussion_r530365320) would prevent the crash, but ideally we should try to understand why we end up with a missing membership at this state, is it a cache issue?

No need to spend too much time investigating if the root cause is not obvious, the issue seems pretty rare.

Reply:
Cache sounds like a good place to start, I'll have a look. Yeah, I haven't ever been able to reproduce locally, even signing up with a fresh account + new collective + going straight into the onboarding modal, which most of these seem to be. A lot of them also seem to be more 'spammy' accounts as well.

I'll keep digging but won't invest too much.